.github/workflows/repo-sync.yml

```diff
@@ -0,0 +1,27 @@
+name: Repo Sync
+
+on:
+  schedule: 
+  - cron: "*/30 06 * * *" # every 15 minutes. set to whatever interval you like
```

Comment:
Update comment to reflect the actual chron value

Reply:
I have updated the comment in the yml file as per the actual schedule time